NetKAN/Arkas-Config-Med-Clouds.netkan

```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
+	"name"		:	"Arkas Med Cloud configuration",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"license"  	:	"CC-BY-NC-SA",
+	"abstract"	:	"Optional cloud support.  This includes the medium level cloud cover",
+	"install"	:	[
+		{
+			"file"		:	"Arkas/Cloud Support/Med Cloud Cover Config/GameData/BoulderCo",
+			"install_to"	:	"GameData"
+		},
+		{
+			"file"		:	"Arkas/Cloud Support/GameData/BoulderCo",
+			"install_to"	:	"GameData"
+		}
+	],
+	"depends"	:	[ { "name"	:	"Arkas" } ],
```

Comment:
I might be wrong and someone can correct me if I am, but I believe all clouds depend on `EnvironmentalVisualEnhancements` unless they write their own .dll which Arkas does not.